create-turbo/.eslintrc.js

```diff
@@ -1,6 +1,26 @@
 module.exports = {
+  root: true,
+  parser: "@typescript-eslint/parser",
+  env: { node: true, jest: true },
+  extends: [
+    "eslint:recommended",
+    "plugin:import/recommended",
+    "plugin:import/typescript",
+  ],
+  plugins: ["import"],
   rules: {
-    // we have an example where we need this
-    "no-undef": "off",
+    "no-empty": ["error", { allowEmptyCatch: true }],
+    "import/no-named-as-default-member": "off",
```

Comment:
Not sure if we want to switch this off, but here are examples of what it prevents from warning:
```
  116:13  warning  Caution: `fse` also has a named export `mkdir`. Check if you meant to write `import {mkdir} from 'fs-extra'` instead          import/no-named-as-default-member
  122:9   warning  Caution: `fse` also has a named export `copy`. Check if you meant to write `import {copy} from 'fs-extra'` instead            import/no-named-as-default-member
  131:11  warning  Caution: `fse` also has a named export `copy`. Check if you meant to write `import {copy} from 'fs-extra'` instead            import/no-named-as-default-member
  144:9   warning  Caution: `fse` also has a named export `move`. Check if you meant to write `import {move} from 'fs-extra'` instead            import/no-named-as-default-member
  162:9   warning  Caution: `fse` also has a named export `writeFile`. Check if you meant to write `import {writeFile} from 'fs-extra'` instead  import/no-named-as-default-member
```